Functions/FirebaseFunctions/FIRFunctions.m

```diff
@@ -31,12 +37,19 @@
 NS_ASSUME_NONNULL_BEGIN
 
 NSString *const kFUNInstanceIDTokenHeader = @"Firebase-Instance-ID-Token";
+NSString *const kFUNDefaultRegion = @"us-central1";
+
+/// Empty protocol to register Functions as a component with Core.
+@protocol FIRFunctionsInstanceProvider
+@end
 
-@interface FIRFunctions () {
+@interface FIRFunctions () <FIRComponentRegistrant, FIRFunctionsInstanceProvider> {
   // The network client to use for http requests.
   GTMSessionFetcherService *_fetcherService;
   // The projectID to use for all function references.
   FIRApp *_app;
+  // The Auth instance to use, may be nil if auth is unavailable.
+  id<FIRAuthInterop> _auth;
```

Comment:
Nit: It doesn't look like we use this var anywhere other than in the constructor, so we can probably get rid of it!

Reply:
Done.